addons/frag/functions/fnc_dev_sphereDraw.sqf

```diff
@@ -0,0 +1,40 @@
+#include "..\script_component.hpp"
+/*
+ * Author: Lambda.Tiger
+ * Add a colored sphere at a specified point.
+ *
+ * Arguments:
+ * 0: ASL position to add sphere. <ARRAY>
+ * 1: Color of sphere. <STRING> (Default: Blue)
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [unit0, player] call ace_frag_fnc_dev_switchUnitHandle;
+ *
+ * Public: No
+ */
+
+params [
+    "_posASL",
+    ["_color", "blue", [""]]
+];
```

Comment:
Is there a downside to specifying it?

Reply:
It's more expensive performance wise, albeit slightly.
More importantly though, when I see an argument requiring a certain type of parameter, it strongly hints towards the function being public.

Reply:
It's equally useful in private use as well. Like, fewer possible bugs being the biggest.

Reply:
I think the performance loss is acceptable in case someone is implementing debugging for a rework of explosion reflections and makes mistakes, but I'll just get rid of it.

Reply:
done by https://github.com/acemod/ACE3/pull/9728/commits/66c566b772c9ff7f7a66a77e86f909b0fcab9f67